src/components/BasicPage.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
+
+import Menu from "../components/Menu";
+import { Logos } from '../components/Logos';
+
+import '../index.css';
+
+export const BasicPage = (props) => {
```

Comment:
I think it would be cool if the logo was above the links, a lot smaller, and a lot more opaque. Would give the page more room to breath. i also still think the logo is a bit much on every page still... i vote for removing personally, but if we have to keep i'm cool if it's moved there.

<img width="1792" alt="Screen Shot 2023-03-27 at 6 17 08 PM" src="https://user-images.githubusercontent.com/17440010/228079688-d242facc-9ac9-470b-872d-d98839d6eb2a.png">